addins/Cake.Mix.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.Mix
```

Comment:
Should be filtered as it is not an addin

Reply:
Done. Cake.Mix added to blacklist